### PR TITLE
Added some additionalPrinterColumns for StorageCluster

### PR DESCRIPTION
**Pull request by nizamial09**
Added three additionalPrinterColumns into the ocsinitialization and storagecluster CRD.
This will generate 3 colomns: `Age`, `Phase` and `Created At` on the ocsinitialization and storagecluster CLI. Screenshots are attached below.

OCSInitialization
ocsinitialization

StorageCluster
storagecluster

Signed-off-by: Nizamudeen <nia@redhat.com>

### CODE REVIEW

**Comment by nizamial09** [comment]
@umangachapagain please review this

File: pkg/apis/ocs/v1/storagecluster_types.go [hunk 1]
```diff
@@ -125,6 +125,8 @@ const (
 // StorageCluster is the Schema for the storageclusters API
 // +k8s:openapi-gen=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=.metadata.creationTimestamp
```
Comment (umangachapagain) [inline pkg/apis/ocs/v1/storagecluster_types.go]:
```suggestion
// +kubebuilder:printcolumn:name="Created At",type=date,JSONPath=.metadata.creationTimestamp
```

**Comment by nizamial09** [comment]
> Can you add screenshots to show how it looks?
> And, update the PR description and Commit Message to clearly state what change is done and why.

Will do that.

**Comment by nizamial09** [comment]
Output of `oc get storagecluster -n openshift-storage`

storagecluster

**Comment by nizamial09** [comment]
Output of `oc get ocsinitialization -n openshift-storage`

ocsinitialization

**Comment by nizamial09** [comment]
@umangachapagain Amended the chages and updated the PR.

**Pull request merged**